components/automate-gateway/api/api_test.go

```diff
@@ -9,15 +9,14 @@ import (
 	_ "github.com/chef/automate/api/external/cfgmgmt"
 	_ "github.com/chef/automate/api/external/common/query"
 	_ "github.com/chef/automate/api/external/common/version"
-	_ "github.com/chef/automate/api/external/nodes"
-	_ "github.com/chef/automate/api/external/nodes/manager"
-	_ "github.com/chef/automate/api/external/secrets"
-	_ "github.com/chef/automate/components/automate-gateway/api/auth/teams"
-	_ "github.com/chef/automate/components/automate-gateway/api/authz"
```

Comment:
Only actual change is deleting this line; introspection endpoints are now included on L24 (not visible here).